README.md

```diff
@@ -31,11 +31,35 @@ public function tools()
 {
     return [
         // ...
-        new \PragmaRX\ArtisanTool\Tool(),
+        new \PragmaRX\ArtisanTool\Tool,
```

Comment:
We should go with: 

```suggestion
        new \PragmaRX\ArtisanTool\Tool(),
```

Because: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#4-classes-properties-and-methods